testsuite/libffi.call/struct_by_value_big.c

```diff
@@ -5,6 +5,7 @@
    Originator:	From the original ffitest.c  */
 
 /* { dg-do run } */
+
```

Comment:
Please remove this unnecessary empty line :).

Reply:
> Please remove this unnecessary empty line :).

My fault. I have deleted this unnecessary empty line. Thanks